sprints/dashboard/tests/test_libs.py

```diff
@@ -0,0 +1,47 @@
+from unittest.mock import patch
```

Comment:
We could place this code in `sprints/dashboard/libs/tests/test_mattermost.py`. What do you think?